lib/schema/operations_user.graphql

```diff
@@ -146,6 +263,167 @@ query FindUserDetailedProfile($userId: String!) {
     }
 }
 
+query FindMentorUserDetailedProfile($id: String!) {
+  findUserById(id: $id) {
+    id
+    firstName
+    lastName
+    fullName
+    offersHelp
+    seeksHelp
+    websites {
+        value
+        label # label == "linkedin" for the linkedin button
```

Comment:
Added some comments about the schema in this file

may need to refactor companies/businessExperiences, not sure about the requirements for that feature